React-Native/en/2.4.4.md

```diff
@@ -0,0 +1,21 @@
+# [Success Criterion 2.4.4 - Link Purpose (In Context)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-refs.html)
+
+In React Native you can use the [Pressable](https://reactnative.dev/docs/pressable) component in combination with the [Linking API](https://reactnative.dev/docs/linking) to create a link. In addition to this you can use the [Text](https://reactnative.dev/docs/text) component to add a link text.
+
+In addition to this it's possible to add extra information about the link purpose by adding one or more of the following properties: [`accessibilityRole`](https://reactnative.dev/docs/accessibility#accessibilityrole), [`accessibilityLabel`](https://reactnative.dev/docs/accessibility#accessibilitylabel) or [`accessibilityHint`](https://reactnative.dev/docs/accessibility#accessibilityhint).
+
+```jsx
+<Pressable
+  onPress={async () => {
+    const supported = await Linking.canOpenURL(url);
+    if (supported) {
+      await Linking.openURL(url);
+    }
+  }}
+  accessibilityRole="link"
+  accessibilityLabel="Link"
```

Comment:
Replace "Link" with "Appt" to avoid having 'Link, link' as announcement when someone would copy paste this.